package.json

```diff
@@ -68,6 +69,7 @@
     "protractor": "~4.0.13",
     "ts-node": "1.2.1",
     "tslint": "^4.3.0",
-    "typescript": "~2.0.3"
+    "typescript": "~2.0.3",
+    "wallaby-webpack": "0.0.31"
```

Comment:
Remove wallaby from the project. While it looks like an awesome project, we are trying to keep the example app simple and focused on the ngrx libraries.